conformance/reports/v0.8.1/cilium-cilium.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+date: "2023-09-25T14:21:25Z"
+gatewayAPIVersion: v0.8.1
+implementation:
+  contact:
+    - https://github.com/cilium/community/blob/main/roles/Maintainers.md
+  organization: cilium
+  project: cilium
+  url: github.com/cilium/cilium
+  version: main
```

Comment:
So this version field is meant to indicate the version of YOUR implementation, just wanted to double check: did you actually want this to be `main` branch, or did you want to associate it with a specific cilium version?

Reply:
I am trying to keep Cilium main branch as closed as possible with Gateway API latest release. As Cilium is having different release cycles, it's only available in main branch. 

If it's not idea to have main version in conformance report, I am happy to wait till we cut the next Cilium release.

Reply:
How long will that be? If that's like days away, then maybe yes it would be better to wait for that, but if that's months out or something maybe just pin to a specific SHA instead of a branch so it's more definitive?

Reply:
There are around 3 more months for Cilium 1.15 dev cycle. Thanks for your pointer, I will update to hash commit shortly.